### PR TITLE
Clone schemas

### PR DESCRIPTION
**Pull request by amit-kleiner**
Without the fix the test would have returned a 400 error for the second POST call. This is because running swagger mutates the schema in the global Fastify instance, and breaks the validation. 
This fix clones the schema when we load it in the Fastify swagger package and prevents mutation.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by amit-kleiner** [comment]
fixed lint, thanks!

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: lib/util/common.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ function addHook (fastify, pluginOptions) {
     // when schemaId is the same in difference instance
     // the latter will lost
     instance.addHook('onReady', (done) => {
-      const allSchemas = instance.getSchemas()
+      const allSchemas = cloner(instance.getSchemas())
```
[the remark this answers is not in the document]
Comment (amit-kleiner) [reply lib/util/common.js:25]: Good question, I copied the package used inside `json-schema-resolver`. 
It only clones the reference object, but doesn't clone the response resolved schema.
Thus the resolved schema contains the original global schema object, which gets mutated.

**Review by climba03003 — APPROVED** [review]
LGTM. Below one just question.

Would it be better if we just clone it once before passing inside json-schema-resolver? I means clone at L38 / L42

**Comment by amit-kleiner** [comment]
> LGTM. Below one just question.
> 
> Would it be better if we just clone it once before passing inside json-schema-resolver? I means clone at L38 / L42



> LGTM. Below one just question.
> 
> Would it be better if we just clone it once before passing inside json-schema-resolver? I means clone at L38 / L42

It would behave the same. I can move it if you think it's better.
Also, should I bump the version to `5.1.1`?

**Comment by amit-kleiner** [comment]
> LGTM. Below one just question.
> 
> Would it be better if we just clone it once before passing inside json-schema-resolver? I means clone at L38 / L42



> LGTM. Below one just question.
> 
> Would it be better if we just clone it once before passing inside json-schema-resolver? I means clone at L38 / L42

It would behave the same. I can move it if you think it's better.
Also, should I bump the version to `5.1.1`?

**Comment by climba03003** [comment]
> It would behave the same. I can move it if you think it's better. 

Yes, please.

> Also, should I bump the version to `5.1.1`?

I will do the release work, no need to change the version inside this PR.

**Pull request merged**